### ISSUE TITLE
Build Instructions

### ISSUE THREAD

**Issue by DefenceLogic** [ComodoSecurity/openedr#3]
Dear Sirs,

Please explain how it is possible to contribute to this project?

I ask because it implies it is "open", however it requires the madhookcode library which you have to purchase at http://www.madcodehook.com/madCodeHookDescription.htm

I will try contacting the developer directory.

**Comment by ComodoMelih** [comment]
if you use the binary we have here, then you don't have to pay for madhookcode as far as I know.

**Comment by pixelsquared** [comment]
@ComodoMelih 
The issue is how can we build and test changes to OpenEDR if we can't build it without a closed source paid library.

I think that the best solution might be to work to remove the madhookcode library as a dependency.

**Comment by ComodoMelih** [comment]
> I think that the best solution might be to work to remove the madhookcode library as a dependency.
agreed! Lets add this to our list.

**Comment by ozercomodo** [comment]
I am trying to arrange some agreement between madhook developer and openEDR so that we can give the code to all our contributors

**Comment by pussinboots1992** [comment]
> I am trying to arrange some agreement between madhook developer and openEDR so that we can give the code to all our contributors

This would be great, it will indeed be hard to contribute to this awesome project without having everything required to build it...

**Comment by trylab** [comment]
NetFilter SDK is also a commercial product.

**Comment by cikgufatah** [comment]
> NetFilter SDK is also a commercial product which was used in the network monitor driver.
Yeah, the some of openedr code referring to non existing function at NetFilter SDK.
Example: https://github.com/ComodoSecurity/openedr/search?q=cmdedr

**Comment by muse117** [comment]
Many source files are missing, resulting in failure to compile.
